.github/workflows/clean.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: install rust
-        uses: risc0/risc0/.github/actions/rustup@release-0.19
+        uses: risc0/risc0/.github/actions/rustup@release-0.19.1
```

Comment:
These can stay at just `release-0.19` since that branch tracks the current patch level release in 0.19